android/gradle.properties

```diff
@@ -18,3 +18,9 @@
 # org.gradle.parallel=true
 
 android.useDeprecatedNdk=true
+
+OLIVER_RELEASE_STORE_FILE=oliver-key.keystore
+OLIVER_RELEASE_KEY_ALIAS=oliver-key
+OLIVER_RELEASE_STORE_PASSWORD=oliverdev
+OLIVER_RELEASE_KEY_PASSWORD=oliverdev
```

Comment:
this info shoulndt be in the repo, remove it and declare it in your global gradle file